apluslms_roman/docker_cache.py

```diff
@@ -0,0 +1,9 @@
+from .cache_file import CacheFile
+
+class DockerCache(CacheFile):
+    name = 'roman_docker_cache'
+    schema = name
```

Comment:
You should add the version here. Note that you can write it as a plain tuple or a string. I prefer tuples for this though. Document metaclass converts it to a Version object.